### ISSUE TITLE
After disabling HA - inspect indicates Service snap.microk8s.daemon-k8s-dqlite is not running

### ISSUE THREAD

**Issue by bondib** [canonical/microk8s#3109]
Reproducing the issue:
`

**snap install microk8s --classic**
microk8s (1.23/stable) v1.23.5 from Canonical✓ installed

**microk8s disable ha-cluster --force**
Reverting to a non-HA setup
Generating new cluster certificates.
Waiting for node to start. .
Enabling flanneld and etcd
HA disabled

**microk8s status**
microk8s is running
high-availability: no
addons:
  disabled:
   ...
   
   
**microk8s inspect**
Inspecting Certificates
Inspecting services
  Service snap.microk8s.daemon-cluster-agent is running
  Service snap.microk8s.daemon-containerd is running
 **FAIL:  Service snap.microk8s.daemon-k8s-dqlite is not running
For more details look at: sudo journalctl -u snap.microk8s.daemon-k8s-dqlite**
  Service snap.microk8s.daemon-kubelite is running
  Service snap.microk8s.daemon-flanneld is running
  Service snap.microk8s.daemon-etcd is running
  Service snap.microk8s.daemon-apiserver-kicker is running
  Copy service arguments to the final report tarball
Inspecting AppArmor configuration
Gathering system information
  Copy processes list to the final report tarball
  Copy snap list to the final report tarball
  Copy VM name (or none) to the final report tarball
  Copy disk usage information to the final report tarball
  Copy memory usage information to the final report tarball
  Copy server uptime to the final report tarball
  Copy current linux distribution to the final report tarball
  Copy openSSL information to the final report tarball
  Copy network configuration to the final report tarball
Inspecting kubernetes cluster
  Inspect kubernetes cluster
Inspecting juju
  Inspect Juju
Inspecting kubeflow
  Inspect Kubeflow
Inspecting dqlite
  Inspect dqlite

Building the report tarball
  Report tarball is at /var/snap/microk8s/3052/inspection-report-20220428_164713.tar.gz

`

.
[inspection-report-20220428_164713.tar.gz](https://github.com/canonical/microk8s/files/8584745/inspection-report-20220428_164713.tar.gz)

**Comment by balchua** [comment]
Yes this is expected.  When HA is disabled, 2 services are started to replace dqlite and calico namely etcd and flannel.  In your logs you will see these

``` console
Service snap.microk8s.daemon-flanneld is running
Service snap.microk8s.daemon-etcd is running
```
Hope it clarifies.

**Comment by bondib** [comment]
Thanks!
Your PR removes the warning which seemed to me like a mistake - so all good now.
Thanks again!
